datafusion/core/src/physical_plan/file_format/parquet/page_filter.rs

```diff
@@ -123,7 +122,7 @@ pub(crate) fn build_page_filter(
     if let (Some(file_offset_indexes), Some(file_page_indexes)) =
         (file_offset_indexes, file_page_indexes)
     {
-        let mut row_selections = VecDeque::with_capacity(page_index_predicates.len());
+        let mut row_selections = Vec::with_capacity(page_index_predicates.len());
```

Comment:
since `row_selections` are always appended to (never pop_front, for example) I think Vec is just fine and using VecDequeue just makes the code more confusing